yarn.lock

```diff
@@ -2237,7 +2237,7 @@ natural-compare@^1.4.0:
   resolved "https://registry.yarnpkg.com/natural-compare/-/natural-compare-1.4.0.tgz#4abebfeed7541f2c27acfb29bdbbd15c8d5ba4f7"
   integrity sha1-Sr6/7tdUHywnrPspvbvRXI1bpPc=
 
-node-fetch@^2.0.0, node-fetch@^2.6.6:
```

Comment:
This other node-fetch will go away after https://github.com/happo/happo.io/pull/344

Reply:
That did not pan out, unfortunately!